packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -242,11 +242,11 @@ class LightCurveViewer extends Component {
       .map((raw) => {
         const x = (raw.minX + raw.maxX) / 2
         const width = Math.abs(raw.maxX - raw.minX)
-        const toolType = props.currentTask.tools[toolIndex].type
+        const toolType = this.props.currentTask.tools[toolIndex].type
```

Comment:
`currentTask` is destructured on line 235. It should be used here. 
```suggestion
        const toolType = currentTask.tools[toolIndex].type
```